Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -2365,7 +2365,7 @@ HTMLInputElement::ValueAttributeMode HTMLInputElement::value_attribute_mode() co
 
 void HTMLInputElement::selection_was_changed(size_t selection_start, size_t selection_end)
 {
-    if (!m_text_node)
+    if (!m_text_node || !document().cursor_position() || !(document().cursor_position()->node() == m_text_node))
```

Comment:
`!(document().cursor_position()->node() == m_text_node)`

->

`document().cursor_position()->node() != m_text_node`

Also in `HTMLTextAreaElement`